### PR TITLE
Map some blaze3d stuff

### PR DESCRIPTION
**Pull request by basaigh**

### CODE REVIEW

File: mappings/com/mojang/blaze3d/platform/GLX.mapping [hunk 1]
```diff
@@ -1,20 +1,32 @@
-CLASS com/mojang/blaze3d/platform/GLX
+CLASS com/mojang/blaze3d/platform/GLX com/mojang/blaze3d/platform/GLX
+	FIELD LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD LOOKUP_MAP Ljava/util/Map;
+	FIELD capsString Ljava/lang/String;
+	FIELD cpuInfo Ljava/lang/String;
+	METHOD _getCapsString ()Ljava/lang/String;
+	METHOD _getCpuInfo ()Ljava/lang/String;
+	METHOD _getLWJGLVersion ()Ljava/lang/String;
```
Comment (modmuss50) [inline mappings/com/mojang/blaze3d/platform/GLX.mapping:8]: Why are these mappings here? They dont have a name or any params?

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping [hunk 2]
```diff
@@ -58,16 +75,45 @@ CLASS com/mojang/blaze3d/systems/RenderSystem
...
+	METHOD disableColorMaterial ()V
+	METHOD disableCull ()V
+	METHOD disableDepthTest ()V
+	METHOD disableFog ()V
+	METHOD disableLighting ()V
+	METHOD disableLineOffset ()V
+	METHOD disablePolygonOffset ()V
+	METHOD disableRescaleNormal ()V
+	METHOD disableScissor ()V
+	METHOD disableTexture ()V
```
Comment (modmuss50) [inline mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:98]: Again here and in the fields, not sure what the point in these are as they dont have a yarn name?

**Review by modmuss50 — CHANGES_REQUESTED** [review]
Something isnt quite right with this PR

File: mappings/com/mojang/blaze3d/systems/RenderSystem.mapping (hunk 2 above)
Comment (basaigh) [reply mappings/com/mojang/blaze3d/systems/RenderSystem.mapping:98]: Oops sorry, don't know what happened there.
I'll fix that when I get time

File: mappings/net/minecraft/SharedConstants.mapping [hunk 3]
```diff
@@ -1,8 +1,9 @@
 CLASS net/minecraft/class_155 net/minecraft/SharedConstants
 	FIELD field_1124 RESOURCE_LEAK_DETECTOR_DISABLED Lio/netty/util/ResourceLeakDetector$Level;
-	FIELD field_1125 isDevelopment Z
+	FIELD field_1125 IS_RUNNING_IN_IDE Z
 	FIELD field_1126 INVALID_CHARS_LEVEL_NAME [C
 	FIELD field_16742 gameVersion Lcom/mojang/bridge/game/GameVersion;
+	FIELD field_22251 MAXIMUMTICKTIMENANOS J
```
Comment (Juuxel) [inline mappings/net/minecraft/SharedConstants.mapping:6]: This should have underscores between the words for readability.

**Comment by YanisBft** [comment]
README is not here for fun. Come with names you thought about next time, not ones you copy over

**Pull request closed without merging**